lnwallet/channel.go

```diff
@@ -3260,7 +3260,7 @@ func (lc *LightningChannel) getUnsignedAckedUpdates() []channeldb.LogUpdate {
 	chanID := lnwire.NewChanIDFromOutPoint(&lc.channelState.FundingOutpoint)
 
 	// Fetch the last remote update that we have signed for.
-	lastRemoteCommitted := lc.remoteCommitChain.tip().theirMessageIndex
+	lastRemoteCommitted := lc.remoteCommitChain.tail().theirMessageIndex
```

Comment:
comment is wrong now?

Reply:
Yeah, I think it should be something like: "the last remote update on their oldest commitment", perhaps?

Reply:
ping @Crypt-iQ

Reply:
Do you agree with the fix besides the comment change? Was deferring until you came to a conclusion

Reply:
Ah, was waiting for the comment to be updated such that I could understand the rationale behind the change a bit better. But yeah, so far the fix looks good, will give it a final look after the comment has been updated.